src/applications/check-in/pre-check-in/tests/e2e/demographics.cypress.spec.js

```diff
@@ -0,0 +1,84 @@
+import '../../../tests/e2e/commands';
```

Comment:
did this file just sneak into this PR? Seems un-related but I'm not mad about it :)

Reply:
Yeah 🥷 it goes with removing the `jumpToPage('complete')` call if demos are complete.